src/platforms/dialogflow/DialogFlowEvent.ts

```diff
@@ -26,6 +26,8 @@ import {
 } from "actions-on-google";
 import { Context as AWSLambdaContext } from "aws-lambda";
 import { Context as AzureContext } from "azure-functions-ts-essentials";
+import { OAuth2Client } from "google-auth-library";
```

Comment:
we should add this to the package.json requirements

Reply:
@armonge done